NetUtils.py

```diff
@@ -94,6 +104,11 @@ class NetworkItem(typing.NamedTuple):
     """ Sending player, except in LocationInfo (from LocationScouts), where it is the receiving player. """
     flags: int = 0
 
+    @classmethod
+    def from_network_dict(cls, source: dict):
+        source.pop("class", None)
+        return cls(**source)
```

Comment:
Have you measured the performance impact of these `from_network_dict` functions? It seems like that could negate a lot of the performance benefit of `orjson`.

But that is not my main concern.
There's a maintenance cost to all of these `from_network_dict` functions. It's a big maintenance cost because it's not type-safe. I think this is not a good solution.

There are good, fast serialization libraries that support `NamedTuple` automatically, without having to maintain functions like this, and they are more type safe.